### ISSUE TITLE
ESC should close "Edit Tag" dialog

### ISSUE THREAD

**Issue by igel-kun** [quodlibet/quodlibet#3358]
The proposed change
-------------------

Quodlibet 4.3.0 (gentoo):
After editing tags, I currently have to close the dialog via the window manager (click on "X"). It would be nice to have a cancel button that reacts to ESC and quits the dialog unless there are pending changes that still need to be applied.

Benefits of this change
-----------------------

Prominently, touchpad users find mouse interactions tiresome, but it seems just good GTK-etiquette to be able to close a windows on button-press

Any issues this might cause
---------------------------

Closing with pending changes should be confirmed by the user.
Closing without pending changes should be fast and easy :)

**Comment by igel-kun** [comment]
Oh this also applies to other dialogs spawned by Quodlibet, like the `Ctrl+I` "Information" dialog...

**Comment by rudluff** [comment]
I would suggest using alt+f4... I use "escape" to cancel an edit to a field without reverting my other changes, so this would result in more clicks for me (escape->no i don't want to cancel all changes just the one->continue editing)

**Comment by igel-kun** [comment]
> I would suggest using alt+f4... I use "escape" to cancel an edit to a field without reverting my other changes, so this would result in more clicks for me (escape->no i don't want to cancel all changes just the one->continue editing)
Good point, but I think GTK has a way of knowing if the user is currently changing the value of an edit field or not.
Regarding your suggestion, `Alt+F4` is cought by my window manager and moves me to desktop#4, but the "close window" command (`Ctrl+W` for me) works, so I guess it's an alternative.

**Comment by CreamyCookie** [comment]
I wish there was some sort of per user `hotkeys` file that application could (must?) comply with.

This way you could set your favorite shortcuts, for example <kbd>Ctrl</kbd> + <kbd>Tab</kbd> = "next Tab", and it would work in "every" application. 

Of course, there will be conflicts and complex programs for which this will not work optimally, but for most applications it would be really useful.

**Comment by Xiretza** [comment]
See also #8, #107, #1556
